ble_controller/doc/README.rst

```diff
@@ -0,0 +1,116 @@
+BLE Controller
```

Comment:
General comment for RST: use one sentence for one line.
See: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html

Reply:
Will keep that in mind for the future.